CNAME

```diff
@@ -1,2 +1 @@
-auroracommunity.co
-www.auroracommunity.co
+eclip.se
```

Comment:
Please don't mess with our CNAME. For testing, use the default GitHub Pages URL format.